src/west/_bootstrap/main.py

```diff
@@ -149,8 +141,8 @@ def init(argv):
 updated.
```

Comment:
The whole documentation block above is outdated and needs to be adapted to the new reality